### DISCUSSION TITLE
Create Incident issue

### DISCUSSION THREAD

**Discussion by ytiruces** [python-gitlab/python-gitlab#2909] [Q&A] [answered]
Hi. How to create Incident?
I think somthing like this:
`issue = project.issues.create({'title': 'I have a bug',
                               'type': 'Incedent'})` ?
I can't find it in docs.

**Comment by JohnVillalovos** [comment]
Your idea looks correct to me.

The docs are here: https://python-gitlab.readthedocs.io/en/main/gl_objects/issues.html#project-issues

As a note they are called "Issues", not "Incidents".

**Reply by ytiruces** [reply, marked as answer]
There is [Incedents](https://docs.gitlab.com/ee/operations/incident_management/incidents.html) object. 
I should try `issue_type` attribute. TY

**Reply by JohnVillalovos** [reply]
I recommend to look at the upstream API documentation then: https://docs.gitlab.com/ee/api/api_resources.html

**Reply by ytiruces** [reply]
![image](https://github.com/python-gitlab/python-gitlab/assets/8374107/50802385-0684-4ffc-8cab-4057ff224014)

**Reply by JohnVillalovos** [reply]
Looks like you have it figured out 👍
